### ISSUE TITLE
1-px-or-so offset when hovering over table elements

### ISSUE THREAD

**Issue by DanielOaks** [ircv3/ircv3.github.io#148]
When you hover over elements, they shift 1px (or-so) to the left. Fix it plz.

**Comment by dequis** [comment]
Didn't we talk about this before?

```
01:01 < dx> dan-: this fixes it for me: .sw-table .support {position: relative} .hover-popup {position: absolute}
```

**Comment by DanielOaks** [comment]
I think I went through a few solutions, but they ended up having various side-effects (or that might've just been me messing with alternate layout options for the bubbles). Thanks, I'll take another squiz at that CSS to see how it goes.

**Comment by DanielOaks** [comment]
While that does fix the issue, Firefox is bugged so we can't do that: https://bugzilla.mozilla.org/show_bug.cgi?id=688556

I'll likely fix it with or after #153

**Comment by xPaw** [comment]
This seems to be fixed?
